ParisBNF/abb/BNFabb28.xml

```diff
@@ -69,7 +69,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <textLang mainLang="gez"/>                                 
                      </msItem>
                      
-                     <msItem xml:id="ms_i1.6">
+                     <msItem xml:id="ms_i2">
```

Comment:
this is wrongly nested and not what I had suggested

Reply:
Ok, sorry, I was in rush. I would use here ms_i1 LIT6608Commentary with five subparts, and ms_i2 for LIT6817CommentJoshua. To be Commentaty on Octateuch, two books are still missing. As far as I understand, only LIT6608Commentary is fixed collection. Its constellation with LIT6817CommentJoshua is casual (abb 28 is the only witness for the latter).

Reply:
Still, judging by the way the texts are copied, the scribe perceived the whole as one collection of Commentaries, therefore I suggested to nest all in one msItem but not create a CAe Id for the collection